Update overview of files and editing (#478)
nhsuk/nhsuk.service-manual.prototype-kit.docs: app/guides/build-basic-prototype/open-prototype-in-editor.md

In your code editor, open your prototype folder. You will see the files and folders that you can edit in your prototype.

> ## Overview of files and folders

Reviewer: Can we update all pages to say ~directory~ folder?

 https://github.com/nhsuk/nhsuk.service-manual.prototype-kit.docs/blob/3c53562005d5785ed0c54f73ab0f0826efe96bff/app/guides/github/store-new-prototype.md?plain=1#L17

Author: will do that as a separate PR

Author: Done in #488

`app` is for your work. Inside that folder:

- `assets` is for CSS, JavaScript, images and downloadable files

> - `data` is where you can put any default data for your prototype

Reviewer: Following the "is for XXX" pattern:

```suggestion
- `data` is for default session data
```

<br>

Does that better explain what it's for?

Author: maybe "session" is jargon at this point?  How about:

> - `data` is for any data that should be loaded by default

Reviewer: What's data?

Author: Good point. How about:

> - `data` is for any default data, such as a pre-selected answer to a question

Reviewer: From the [**Passing data page to page** guidance](https://prototype-kit.service-manual.nhs.uk/guides/passing-data-page/) we say:

>When a user answers questions, their answer is saved within the ‘data’ object using the name attribute of the input.

Maybe change "any default data" to "saving defaults"?

```suggestion
- `data` is for saving defaults, for example answers to questions
```

Reviewer: Could even mention that defaults stay around when clicking 'Reset data'

> - `views` is where the templates for each of your pages go – this is where you’ll spend most of your time editing

Reviewer: ```suggestion
- `views` is for page templates – this is where you’ll spend most of your time editing
```

Outside of a folder are some additional files:

- `config.js` is used for configuring some settings

> - `filters.js` can be used to add your own helpful filters, but this is not covered in this tutorial

Reviewer: Are all the others `config.js`, `locals.js` and `README.md` definitely covered in the tutorial, making this file the odd one out?

```suggestion
- `filters.js` can be used to add your own helpful filters
```

- `locals.js` can be used to set any information you’d like to be able to include on any page, for example the name of your organisation
- `routes.js` is for advanced logic – for example, if a user should go to one page or another based on their answers (we'll cover this later)
- `README.md` can be used to add any notes about your prototypes for yourself or others working on it

## Editing files

> In most code editors, you can toggle folders open or closed, and click files to open them in a separate panel.

Reviewer: ```suggestion
In most code editors you can toggle folders open or closed, and click files to open them in a separate panel.
```

You can have more than one file open at once in tabs, and close them when you are finished editing them.

> Depending on your editor and its settings, you may need to press ‘Save’ (Control-S on Windows or Command-S on a Mac) to save changes. In Codespaces this is set up to save automatically, although you can turn this feature off later if you prefer.

Reviewer: Want to use the [new nested `<kbd>` global styles](https://nhsuk.github.io/nhsuk-frontend/examples/code-blocks/) for these?

Do most Windows keyboard shorten Control → Cntrl?

Noticed it's what GitHub does
https://docs.github.com/en/get-started/accessibility/keyboard-shortcuts

Author: @colinrotherham ooh, didn't know this was a thing! Also means we can delete some custom CSS we had for this. See #487

Author: This is done now.
